app/components/yearGraph/index.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { FC, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { AppState } from '../../reducers';
+import { Year } from '../../model/aggregation';
+import formatNumber from '../../helpers/formatNumber';
+const s = require('./yearGraph.scss');
+const useStyles = require('isomorphic-style-loader/useStyles');
+
+const YearGraph: FC = () => {
+  useStyles(s);
+  const [currentYearSet, setCurrentYearSet] = useState<Year | null>(null);
+
+  const allYears = useSelector((state: AppState) => state.searchFilterState.yearAll);
+
+  const maxWidth = 180; // px
+  const maxHeight = 100; // px
```

Comment:
How about pass these variables as props and set a default value?

Reply:
good point. I forgot about it. that was first idea.